pyiode/objects/equation.pyx

```diff
@@ -116,6 +116,28 @@ cdef class Equation:
         wrapper.ptr_owner = owner
         return wrapper
 
+
+    def _format_lec_string(self, lec_definition: str) -> str:
+        """
+        Format the LEC string by aligning it symmetrically around ':=' and indenting (potential) multi-lines.
+
+        Parameters
+        ----------
+        lec_definition: str
+            The LEC string to format.
+
+        Returns
+        -------
+        str
+            The formatted LEC string.
+        """
+        left_side, right_side = lec_definition.split(':=')[0].strip(), lec_definition.split(':=')[1].strip()
```

Comment:
It would be better to not repeat code.
I don't know where call this function exactly but would it be possible to call `Equation.split_equation()` here ? 
Or make `_format_lec_string()` and `Equation.split_equation()` to share the repeated code -> shared code in pyiode/iode/utils.py ?

Reply:
The following one-liner via list comprehension might also do the trick: `left_side, right_side = [part.strip() for part in lec_definition.split(':=')]`

Reply:
I have checked and LEC expressions do not always contain a `:=`. 
For instance, Identities are defined by LEC expressions without `:=`. 
So:
1. Move your function `_format_lec_string` should be moved to pyiode\util.pyx
2. You need to test if the passed LEC expression contains `:=`
3. If it does: 
```python
    left_side, right_side = [part.strip() for part in lec_definition.split(':=')]
    right_side = right_side.replace('\n', f'\n{lec_indent}')
    lec_definition = f"{left_side} := {right_side}"
```
5. If not:
```python
    lec_definition = lec_definition.strip().replace('\n', f'\n{lec_indent}')
```
6. Finish by:
```python
    return lec_definition 
```

Then call `_format_lec_string` in:
1. pyiode\objects\equation.pyx
2. pyiode\iode_database\identities_database.pyx in method `def _get_object(self, key: str)`

Also update examples involving identites if needed (but I don't think there is a `\n` in any identities in the "fun" model)

Reply:
`format_lec_string` merely gives indents (`lec_indent`) for follow-up lines when given a multi-line definition. These indents correspond to (the size of) the left hand term `A` of the lec definition (`A:=B\nC\nD`).  But this logic does not really apply to identities, for they have no left hand term to begin with. Things like `io.identities['UCLASS']` just give: 

`NATY-(NDOMY+NIY+NGY+EFXY-EFMY+(KNFY*KLFHP)/(0.001*HF*(1-(1-\nPROF)*0.01*PRDAL)))`

So, I only moved the function to `pyiode\util.pyx` but have not applied it to the case of identities.